_data/training.yml

```diff
@@ -246,5 +246,13 @@
   date: 2021-07-05
   description: Part 1 covers class design patterns, the python memory model, debugging, profiling, and more. Part 2 covers Python features and packaging. Part 3 covers common Python packages.
   tags: {Python}
+- name: "Managing-research-software-projects"
+  url: https://swcarpentry.github.io/managing-research-software-projects/
+  author: Daniel Standage, Greg Wilson
+  author_url: https://github.com/swcarpentry/managing-research-software-projects
+  location: Software Carpentry Foundation
+  location_url: https://software-carpentry.org/blog/archive/
```

Comment:
```suggestion
```

Reply:
In this case, because it's not associated with a specific training event, I think we should just leave these three fields out.